modules/net/examples/ping_pong.cpp

```diff
@@ -0,0 +1,110 @@
+//=================================================================================================
+// Copyright (C) 2023-2025 HEPHAESTUS Contributors
+//=================================================================================================
+
+#include <array>
+#include <chrono>
+#include <exception>
+#include <memory>
+#include <span>
+#include <string_view>
+#include <utility>
+
+#include <exec/async_scope.hpp>
+#include <exec/task.hpp>
+#include <fmt/base.h>
+#include <fmt/format.h>
+#include <hephaestus/concurrency/context_scheduler.h>
+#include <hephaestus/net/endpoint.h>
+
+#include "hephaestus/cli/program_options.h"
+#include "hephaestus/concurrency/context.h"
+#include "hephaestus/net/accept.h"
+#include "hephaestus/net/acceptor.h"
+#include "hephaestus/net/recv.h"
+#include "hephaestus/net/send.h"
+#include "hephaestus/net/socket.h"
+#include "hephaestus/telemetry/log.h"
+#include "hephaestus/telemetry/log_sinks/absl_sink.h"
+
+namespace {
+inline constexpr unsigned PAGE_SIZE = 4096;
+// NOLINTNEXTLINE (readability-static-accessed-through-instance)
```

Comment:
why is this needed?

Reply:
There seems to be a issue within the task implementation which access a static variable through an object and clang-tidy starts to complain.